README.md

```diff
@@ -240,25 +240,22 @@ csi-secrets-store-provider-azure-sxht2             2/2     Running   0
 
 ## Providers
 
-This project features a pluggable provider interface developers can implement that defines the actions of the Secrets Store CSI driver.
+This project features a pluggable provider interface developers can implement that defines the actions of the Secrets Store CSI driver. This enables retrieval of sensitive objects storied an enterprise-grade external secrets store into Kubernetes while continue to manage these objects outside of Kubernetes.
```

Comment:
Should it be `This enables retrieval of sensitive objects stored in enterprise-grade external secret stores into Kubernetes`? Do we need to mention `managed these objects outside of kubernetes`?

Reply:
I think it is useful to contrast this approach with Kubernetes secrets in this way

Reply:
nit: `storied in` to `stored in`